pkg/kyverno/apply/Readme.md

```diff
@@ -0,0 +1,9 @@
+Apply Command - Valid Parameter Combinations and their interpretation!
+
+| S.No | policy        | resource         | cluster   | namespace      | interpretation                                                                           |
+| ---- |:-------------:| :---------------:| :--------:| :-------------:| :----------------------------------------------------------------------------------------| 
+| 1.   | policy.yaml   | -r resource.yaml | false     |                | apply policy from file to the resource from file                                         |
+| 2.   | policy.yaml   | -r resourceName  | true      |                | apply policy from file to the resource in cluster                                        |
+| 3.   | policy.yaml   |                  | true      |                | apply policy from file to all the resources in cluster                                   |
+| 4.   | policy.yaml   | -r resourceName  | true      | -n=namespace   | apply policy from file to the resource in cluster in mentioned namespace                 |
+| 5.   | policy.yaml   |                  | true      | -n=namespace   | apply policy from file to all the resources in cluster in mentioned namespace            |
```

Comment:
I tested locally, 1, 3 worked as expected, got `POLICY REPORT: not generated as no validation failure` for 2, 4, 5.

Am I using the right command?
```
kyverno apply test.yaml  --policy_report --cluster -r nginx-2-5fc7bcd4d9-49lph -n default
```

Reply:
Yes, the command is correct. 

Can you share the test.yaml and resource manifest for which you are testing.